packages/booter-lb3app/src/lb3app.booter.component.ts

```diff
@@ -0,0 +1,20 @@
+import {BootBindings, Bootstrapper} from '@loopback/boot';
+import {
+  Application,
+  BindingScope,
+  Component,
+  CoreBindings,
+  inject,
+} from '@loopback/core';
+import {Lb3AppBooter} from './lb3app.booter';
+
+export class Lb3AppBooterComponent implements Component {
+  booters = [Lb3AppBooter];
+
+  constructor(@inject(CoreBindings.APPLICATION_INSTANCE) app: Application) {
+    app
+      .bind(BootBindings.BOOTSTRAPPER_KEY)
+      .toClass(Bootstrapper)
+      .inScope(BindingScope.SINGLETON);
```

Comment:
We should not bind the `BOOTSTRAPPER_KEY` here in this extension. Please remove it.